__tests__/integration/runner.js

```diff
@@ -60,6 +54,10 @@ for (const testName of readdirSync(__dirname)) {
         // NOTE: We only compare files so that we give the test runner a more detailed diff.
         return;
       }
+      // NOTE: Skip source map on Windows, because generated source map has different newline code (CRLF).
```

Comment:
Skipping source map on Windows, because generated source map has different newline code (CRLF) to expected one on Windows.
I think the exact test of this is a responsibility of Rollup.

Reply:
Hmm, stripWhitespace() was supposed to handle this. It was meant to strip all whitespace, including CRLF.

 Can we try a different regex in stripWhitespace()? If that doesn't work, then adding this fallthrough is fine.

```
stdout.replace(/[ \t\r\n\f]+$/gm, '');
```

Reply:
@FredKSchott The problem was the escaped newline included in json value like '\\r\\n` or '\\n'.
Stripping it in https://github.com/pikapkg/snowpack/pull/178/commits/9024577c6dbe69d66a16cdea6a10ddc3ebacbbe3, please review!

Reply:
Great! Thanks for putting in the extra time so that we get full coverage!